js-foundations.js

```diff
@@ -33,10 +65,22 @@ var testString2 = "academy"
 var testString3 = "sandiego"
 
 // 4a. Write a function called alphabetSoup that takes one argument and returns a string with all the letters in alphabetical order. Use each of the varibales as test cases. Expected output: "aelnr" "aacdemy" "adeginos"
+let alphabetSoup = function(alpha){
+let arr = alpha.split("")
+  return arr.sort().join('')
+}
 
+console.log(alphabetSoup(testString1))
 
 
 // 4b. Write a function that takes in all three variables and returns a string with all the letters in alphabetical order. Expected output: "aaaacddeeegilmnnorsy"
+let alphabetSoup = function(alpha){
+let arr = alpha.split("")
+let newArr = []
+  return arr.sort().join('').push(newArr)
+}
+
+console.log(alphabetSoup(testString1,testString2,testString3))
 
 
```

Comment:
If you are going to pass in all three variables, you need your function to be able to accept all the arguments.